src/libs/ReportUtils.js

```diff
@@ -3940,6 +3941,15 @@ function shouldDisableRename(report, policy) {
     return !_.keys(loginList).includes(policy.owner) && policy.role !== CONST.POLICY.ROLE.ADMIN;
 }
 
+/**
+ * @param {Object|null} report
+ * @param {Object|null} policy - the workspace the report is on, null if the user isn't a member of the workspace
+ * @returns {Boolean}
+ */
+function canEditWriteCapability(report, policy) {
```

Comment:
Does the API already check for this? Or are we only checking this on the front end?